boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -0,0 +1,284 @@
+.. _mec15xxevb_assy6853:
+
+MEC15xxEVB ASSY6853
+###################
+
+Overview
+********
+
+The MEC15xxEVB_ASSY6853 kit is a future development platform to evaluate the
+Microchip MEC15XX series microcontrollers. This board needs to be mated with
+part number MEC1501 144WFBA SOLDER DC ASSY 6860(cpu board) in order to operate.
+
+.. image:: ./mec15xxevb_assy6853.jpg
+     :width: 500px
+     :align: center
+     :alt: MEC15XX EVB ASSY 6853
+
+Hardware
+********
+
+- MEC1501HB0SZ ARM Cortex-M4 Processor
+- 256 KB Ram and 64 KB boot Rom
+- Keyboard interface
+- ADC & GPIO headers
+- UART0, UART1, and UART2
+- FAN0, FAN1, FAN2 headers
+- FAN PWM interface
+- JTAG/SWD, ETM and MCHP Trace ports
+- PECI interface 3.0
+- I2C voltage translator
+- 10 SMBUS headers
+- 4 SGPIO headers
+- VCI interface
+- 5 independent Hardware Driven PS/2 Ports
+- eSPI header
+- 3 Breathing/Blinking LEDs
+- 2 Sockets for SPI NOR chips
+- One reset and VCC_PWRDGD pushbuttons
+- One external PCA9555 I/O port with jumper selectable I2C address.
+- One external LTC2489 delta-sigma ADC with jumper selectable I2C address.
+- Board power jumper selectable from +5V 2.1mm/5.5mm barrel connector or USB Micro A connector.
+
+For more information about the SOC please visit:
```

Comment:
Better would be to join these lines into:
```
For more information about the SoC please see the `MEC1501 Reference Manual`_
```

Reply:
Thanks. Done in 1476db16